src/core/layout/qgslayoutitemattributetable.cpp

```diff
@@ -368,16 +367,16 @@ void QgsLayoutItemAttributeTable::restoreFieldAliasMap( const QMap<int, QString>
     return;
   }
 
-  for ( QgsLayoutTableColumn *column : qgis::as_const( mColumns ) )
+  for ( QgsLayoutTableColumn &column : mColumns )
```

Comment:
My understanding is that you're NOT allowed to modify a qt container inside a range based loop. Can you confirm that this is safe?

Reply:
correct, I will fix this